### PR TITLE
app-emulation/virtio-win updated to 173 and added new proxy-maintainer

### PR DESCRIPTION
**Pull request by epsilon-0**
Updated an old package for KVM Windows drivers.

Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: app-emulation/virtio-win/virtio-win-0.1.173.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="VirtIO drivers for Windows virtual machines running on KVM"
+HOMEPAGE="https://fedoraproject.org/wiki/Windows_Virtio_Drivers"
```
Comment (juippis) [inline app-emulation/virtio-win/virtio-win-0.1.173.ebuild:7]: This homepage seems outdated.

File: app-emulation/virtio-win/virtio-win-0.1.173.ebuild [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+SRC_URI="https://fedorapeople.org/groups/virt/${PN}/direct-downloads/archive-virtio/${PN}-${PV}-9/${PN}-${PV}.iso"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+S=${WORKDIR}
```
Comment (juippis) [inline app-emulation/virtio-win/virtio-win-0.1.173.ebuild]: `S="..."`

File: app-emulation/virtio-win/virtio-win-0.1.173.ebuild (hunk 1 above)
Comment (epsilon-0) [reply app-emulation/virtio-win/virtio-win-0.1.173.ebuild]: Thanks for noticing, I've added the new page.

The url is a long one, hopefully it's not a problem.

File: app-emulation/virtio-win/virtio-win-0.1.173.ebuild (hunk 2 above)
Comment (epsilon-0) [reply app-emulation/virtio-win/virtio-win-0.1.173.ebuild]: I've added the quotes.

**Comment by epsilon-0** [comment]
> You need to follow our git commit syntax, or use repoman that automatically prepends the correct values.
> https://www.gentoo.org/glep/glep-0066.html#commit-messages

Thanks a lot, that was pretty informative. I've updated the commit messages accordingly.

I have another small change I want to make but it requires some shell scripting.
I'll add it and you can see if it is a good thing to keep.

**Pull request closed without merging**